src/Containers/CRMTable/CRMTable.tsx

```diff
@@ -62,6 +66,11 @@ export const CRMTable: React.FC<ICRMTableProps> = ({
             ])
         })
 
+    const justMenuClick = (event: React.MouseEvent<SVGElement, MouseEvent>) => {
```

Comment:
onMenuClick?
JSDocs missing

Reply:
onMenuClick JSDoc is on line 15, or does justMenuClick need one as well?